modules/ding_loan/plugins/content_types/loans.inc

```diff
@@ -289,6 +289,10 @@ function ding_loan_loans_form_submit($form, &$form_state) {
       $title = $entity->getTitle();
     }
 
+    if (!empty($form_state['values']['items'][$id]->notes) && !($form_state['values']['items'][$id]->ill_status)) {
```

Comment:
Please add a code comment describing why this logic is needed.

The issue deals with periodicals, the code checks for notes and inter library loan status and then only prints notes.